packages/firestore/src/remote/datastore.ts

```diff
@@ -246,12 +246,17 @@ export async function invokeRunAggregationQueryRpc(
     datastoreImpl.serializer,
     queryToTarget(aggregateQuery.query._query)
   );
+
+  //while invoking streaming RPC, grpc_connection requires parent in the request
```

Comment:
Please re-write this comment as a "command" to the program (e.g. "Include the parent in the request for grpc_connection...".

Wording comments this way leads to more readable inline comments.

Also, write it as a complete, valid English sentence. So start the sentence with an uppercase letter and end it with a period.

Finally, Add a space between `//` and the first letter of the comment.